### PR TITLE
Add v1.1 change log

### LINKED ISSUE #570: Create change log for 1.1
We should have a concise description of the changes from 1.0 to 1.1. Something like https://iiif.io/api/image/3.0/change-log/

### PR DESCRIPTION
**Pull request by zimeon**
Fixes #570

### CODE REVIEW

**Comment by zimeon** [comment]
Some additional changes are required to link this in to the 1.1 spec etc.. And, of course, when this is published we'll need to change `/draft/` to `/1.1/` in many URIs.

File: draft/spec/change-log.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+This document is licensed under a [Creative Commons Attribution 4.0
+License](https://creativecommons.org/licenses/by/4.0/). [OCFL logo:
+"hand-drive"](https://avatars0.githubusercontent.com/u/35607965) by
+[Patrick Hochstenbach](http://orcid.org/0000-0001-8390-6171) is
+licensed under [CC BY 2.0](https://creativecommons.org/licenses/by/2.0/).
+
+## Changes from OCFL v1.0 to v1.1
+
+[Version 1.1 of the OCFL Specification](https://ocfl.io/draft/spec/) is a [minor version](https://semver.org/) update to the [OCFL Specification v1.0](https://ocfl.io/1.0/spec/). It is intended to add functionality in a backwards compatible manner, and to clarify certain aspects based on a community feedback.
```
Comment (awoods) [inline draft/spec/change-log.md:26]: Suggestion from:
```
, and to clarify certain aspects based on a community feedback
```
to:
```
by limiting changes to correction and clarification.
```

File: draft/spec/change-log.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+### Additions in v1.1
+
+#### Add requirements version number sequence
+
+Added [Conformance of prior versions](https://ocfl.io/draft/spec/#conformance-of-prior-versions) section to clarify that existing version directories are immutable and that the version number sequence must be monaotonic. Adds error code [E103](https://ocfl.io/draft/spec/#E103). (Issue [#544](https://github.com/OCFL/spec/issues/544))
```
Comment (awoods) [inline draft/spec/change-log.md:32]: Typo: `monaotonic` -> `monotonic`

File: draft/spec/change-log.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+[Patrick Hochstenbach](http://orcid.org/0000-0001-8390-6171) is
+licensed under [CC BY 2.0](https://creativecommons.org/licenses/by/2.0/).
+
+## Changes from OCFL v1.0 to v1.1
+
+[Version 1.1 of the OCFL Specification](https://ocfl.io/draft/spec/) is a [minor version](https://semver.org/) update to the [OCFL Specification v1.0](https://ocfl.io/1.0/spec/). It is intended to add functionality in a backwards compatible manner, and to clarify certain aspects based on a community feedback.
+
+### Additions in v1.1
+
+#### Add requirements version number sequence
```
Comment (awoods) [inline draft/spec/change-log.md:30]: Suggestion from:
```
Add requirements version number sequence
```
to:
```
Add requirements to version number sequence
```

File: draft/spec/change-log.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+#### Clarify manifest block requirements
+
+Add language in [Manifest](https://ocfl.io/draft/spec/#manifest) section to clarify that the `manifest` block must be a JSON object (adding error code [E106](https://ocfl.io/draft/spec/#E106)) and that the each key must correspond to a digest value key found in one or more `state` blocks (adding error code [E107](https://ocfl.io/draft/spec/#E107)). (Issue [#537](https://github.com/OCFL/spec/issues/537))
+
+#### Clarify manifest requirements in historic inventories
+
+Wording of the [Content Directory](https://ocfl.io/draft/spec/#content-directory) section improved to make it clear that for each historical inventory, the manifest must reference every file in that version directory. (Issue [#538](https://github.com/OCFL/spec/issues/538))
+
+#### Clarify language and error codes for version numbers
+
```
Comment (awoods) [inline draft/spec/change-log.md:57]: Suggestion from:
```
Change [Version Directories](https://ocfl.io/draft/spec/#version-directories) section changed to be more
```
to:
```
Change [Version Directories](https://ocfl.io/draft/spec/#version-directories) section to be more
```

File: draft/spec/change-log.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+Update [Basic Structure](https://ocfl.io/draft/spec/#inventory-structure) section to make it clear that the `id` must not change between versions of the same object. Adds error code [E110](https://ocfl.io/draft/spec/#E110). (Issue [#542](https://github.com/OCFL/spec/issues/542))
+
+#### Use logical state consistently
+
+Use the notion of "logical state" consistently in the [Version](https://ocfl.io/draft/spec/#version), [Version Inventory and Inventory Digest](https://ocfl.io/draft/spec/#version-inventory) and [BagIt in an OCFL Object](https://ocfl.io/draft/spec/#example-bagit-in-ocfl) sections. (Issue [#571](https://github.com/OCFL/spec/issues/571))
+
+#### Clarify digest value case sensitivity requirements
+
+Change [Manifest](https://ocfl.io/draft/spec/#manifest) and [Fixity](https://ocfl.io/draft/spec/#fixity) sections to make it clear that the additional requirement for each digest value to appear only once in the manifest or fixity block applies only in the case of case-insensitive digest algorithms. (Issue [#573](https://github.com/OCFL/spec/issues/573))
```
Comment (awoods) [inline draft/spec/change-log.md:74]: Suggestion from:
```
applies only in the case of case-insensitive digest algorithms
```
to:
```
applies only to case-insensitive digest algorithms
```

File: draft/spec/change-log.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+Even for minor releases the validations codes may be updated. We have thus moved the `validation-codes.md` file into each version directory so that will be versioned along with the specification. The version of this file for the v1.1 specification is rendered as <https://ocfl.io/draft/spec/validation-codes.html>. (Issue [#553](https://github.com/OCFL/spec/issues/553))
+
+#### Fix E048 description
+
+The E048 error description in `validation-codes.md` is corrected to remove mention of `message` and `user` because they are optional. (Issue [#531](https://github.com/OCFL/spec/issues/531))
+
+#### Fix E070 description
+
+The E070 error description in `validation-codes.md` is corrected to refer to `extension` rather and `key` (which was left from an earlier draft). (Issue [#573](https://github.com/OCFL/spec/issues/573))
```
Comment (awoods) [inline draft/spec/change-log.md:108]: Typo:
```
refer to `extension` rather and `key`
```
->
```
refer to `extension` rather than `key`
```

**Review by awoods — CHANGES_REQUESTED** [review]

File: draft/spec/change-log.md (hunk 2 above)
Comment (zimeon) [reply draft/spec/change-log.md]: Thanks, fixed in [e7a7b38](https://github.com/OCFL/spec/pull/610/commits/e7a7b38655b195fdc3351785dc27bed35a8f5ff0)

File: draft/spec/change-log.md (hunk 3 above)
Comment (zimeon) [reply draft/spec/change-log.md]: Thanks, fixed in [e7a7b38](https://github.com/OCFL/spec/pull/610/commits/e7a7b38655b195fdc3351785dc27bed35a8f5ff0)

File: draft/spec/change-log.md (hunk 4 above)
Comment (zimeon) [reply draft/spec/change-log.md:57]: Thanks, fixed in [e7a7b38](https://github.com/OCFL/spec/pull/610/commits/e7a7b38655b195fdc3351785dc27bed35a8f5ff0)

File: draft/spec/change-log.md (hunk 5 above)
Comment (zimeon) [reply draft/spec/change-log.md]: Thanks, fixed in [e7a7b38](https://github.com/OCFL/spec/pull/610/commits/e7a7b38655b195fdc3351785dc27bed35a8f5ff0)

File: draft/spec/change-log.md (hunk 6 above)
Comment (zimeon) [reply draft/spec/change-log.md]: Thanks, fixed in [e7a7b38](https://github.com/OCFL/spec/pull/610/commits/e7a7b38655b195fdc3351785dc27bed35a8f5ff0)

File: draft/spec/change-log.md (hunk 1 above)
Comment (zimeon) [reply draft/spec/change-log.md:26]: I think we are adding one new thing which is the idea of prior version conformance so it wouldn't be true to say strictly corrections and clarification.  How about:

> The focus is correction and clarification, plus the addition of backwards compatible rules for the conformance of prior object versions and the associated specification version number sequence.

**Review by awoods — APPROVED** [review]

File: draft/spec/change-log.md (hunk 1 above)
Comment (awoods) [reply draft/spec/change-log.md:26]: Looks good.

Comment (zimeon) [reply draft/spec/change-log.md]: Fixed in [f53c4f6](https://github.com/OCFL/spec/pull/610/commits/f53c4f6055fed95dd6838eca8d1a9877f93c224a)

**Review by rosy1280 — APPROVED** [review]

**Comment by rosy1280** [comment]
way better than my first pass.  thanks @zimeon

**Review by julianmorley — APPROVED** [review]

**Pull request merged**